### PR TITLE
Change shift-click customization behaviour

### PR DESCRIPTION
**Pull request by Kamielvf**
This will change the current behaviour to the proposed one which was discussed on discord.

Right click inventory icon and enable shift-click customization.
Right click an item and select the preferred shift-click action, or reset to set it to its default.

[preview](https://puu.sh/zUjYa/1b90d2c3c0.gif)

### CODE REVIEW

File: runescape-client/src/main/java/Client.java [hunk 1]
```diff
@@ -6224,6 +6224,9 @@ final boolean method1211() {
       garbageValue = "13"
    )
    @Export("openMenu")
+   @Hook(
```
Comment (Adam-) [inline runescape-client/src/main/java/Client.java]: Is using the field isMenuOpen not sufficient for this?

Comment (Kamielvf) [reply runescape-client/src/main/java/Client.java]: I could but I'd prefer to only fire the event when the menu is actually opened and not when the field changes from true to false.

Comment (Adam-) [reply runescape-client/src/main/java/Client.java]: so, check that it changes to true.

Comment (Kamielvf) [reply runescape-client/src/main/java/Client.java]: oh.. :man_facepalming:

**Pull request merged**